README.md

```diff
@@ -24,7 +24,7 @@ If your team uses Tock and Slack, you might also find the ["angrytock" reminder
   $ docker-compose build
   $ docker-compose run app python manage.py migrate
   $ docker-compose run app python manage.py loaddata test_data/data-update-deduped.json
-  $ docker-compose run app python manage.py createsuperuser --username admin@gsa.gov --email admin@gsa.gov --noinput
+  $ docker-compose run app python manage.py createsuperuser --username admin --email admin@gsa.gov --noinput
```

Comment:
@Jkrzy, What about making it: _new lines **just** for readability here_

```sh
docker-compose run app \
    python manage.py \
    createsuperuser \
    --username admin.user \
    --email admin.user@gsa.gov \
    --noinput
```

Reply:
Great thought! That would stick out as abnormal on the employees URL path.